.github/workflows/05_nightly.yaml

```diff
@@ -0,0 +1,31 @@
+name: nightly
+
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ci:
+    uses: ./.github/workflows/100_ci.yml
+    permissions:
+      packages: write
+      contents: write
+      security-events: write
+    secrets: inherit
+    with:
+      unit_tests: skip
+      sast: skip
+      docs: skip
+      integration_tests: skip
+      output_type: table
+
+  cleanup-registry:
```

Comment:
I would assume this job needs more permissions than none, but maybe I'm mistaken

Reply:
It's the same as before and that works 🤷 I guess since a separate token is passed on, which has enough permissions there is no need to define additional `permissions`.